bin/acli

```diff
@@ -17,6 +17,7 @@
 namespace Acquia\Cli;
 
 use Acquia\Cli\Command\Api\ApiCommandHelper;
+use Acquia\Cli\EventListener\ExceptionListener;
```

Comment:
This include shouldn't be necessary, should it?

Reply:
Correct, this is an artifact from a previous iteration.